### PR TITLE
fix(config): add missing params to params.ts

### PR DESCRIPTION
**Pull request by jmcook1186**
### Types of changes

- [ ] Enhancement (project structure, spelling, grammar, formatting)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### A description of the changes proposed in the Pull Request
Add missing params to params.ts

### CODE REVIEW

**Comment by narekhovhannisyan** [comment]
@jmcook1186 shouldn't we add instance type too?

**Comment by jmcook1186** [comment]
`cloud/instance-type` is already there

**Comment by narekhovhannisyan** [comment]
@jmcook1186 ok in that case we should update manifests

**Review by narekhovhannisyan — APPROVED** [review]

**Review by manushak — APPROVED** [review]

File: examples/manifests/nesting.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ tree:
     child-0:
       defaults:
         cpu/thermal-design-power: 100
-        grid/carbon-intensity: 800
+        cloud/grid/carbon-intensity: 800
```
Comment (narekhovhannisyan) [inline examples/manifests/nesting.yml:53]: here

Comment (jmcook1186) [inline examples/manifests/nesting.yml:53]:
```suggestion
       grid/carbon-intensity: 800
```

**Review by MariamKhalatova — APPROVED** [review]

**Pull request merged**